### ISSUE TITLE
Focus syles are applied to scheduler elements in chrome

### ISSUE THREAD

**Issue by joneff** [telerik/kendo-themes#2169]
@joneff 
I now see similar behavior in the Scheduler component:
https://winslaveblazor.dev.progress.com/blazor-ui-production/scheduler/overview

![image](https://user-images.githubusercontent.com/5861835/93207712-0ae82e00-f764-11ea-9455-849b1cb6f70c.png)

_Originally posted by @matanasov in https://github.com/telerik/kendo-themes/issues/1796#issuecomment-692668577_

**Comment by Dimitar-Goshev** [comment]
@joneff In Kendo UI for jQuery currently there is no focus indicator, when the k-nav-current element (the dates in the Scheduler header) is focused. Losing current focused item indication is an accessibility issue for Kendo UI for jQuery Scheduler, which allows moving focus over the tools in the header.
